### ISSUE TITLE
Need some explanation for a cryptic error message

### ISSUE THREAD

**Issue by andreashandel** [mlr-org/mlr#1791]
Hi,
I'm doing feature selection for a linear model (using learner lm) with GA and 10 fold CV 10 times repeated. For one of my datasets, I keep getting the following error message:

Error in solve.default(covg[diese, diese], matrix(covg[diese, andere],  : 
  'a' is 0-diml

Unfortunately, I have no idea what that wants to tell me. I suspect it might have to do with a value of a factor showing up in the test set during CV that was not part of the training, but that's just a guess. Could someone give me a pointer as to what this error might mean? (Unfortunately I'm not able to provide a reproducible example, I can only make the error happen for that dataset, which I can't share).

**Comment by berndbischl** [comment]
at least i can tell you that this is not generated by mlr, but from somewhere inside of lm.
can you try to see if it happens if you fit a model with 0 features?

**Comment by berndbischl** [comment]
also pretty funny that someone apparentlly uses german var names in the code....

**Comment by berndbischl** [comment]
well. it also took 5 secs to google at leats parts of the puzzle

http://stackoverflow.com/questions/22704934/running-error-within-function-using-solve-in-r

**Comment by berndbischl** [comment]
ok what *would* help is this:

a) a full traceback

b) the log from your run - if possible

c) a minimal piece of code which is the same what you run, but with other data

**Comment by andreashandel** [comment]
Thanks for the quick reply. I also found that stackoverflow question and since the error message involved German variable names, I figured mlr is sending something to solve.default which the latter doesn't like. 
Unfortunately, my error is annoying insofar that it is intermittent. In my independent data cleaning step, I run imputation, and it seems that sometimes, the cleaned dataset is such that the error does not show up, other times it does. Right now error is gone again... If/when it shows up again, I'll try to do my best and 'catch' it in a maybe reproducible manner.
